apps/web/src/app/[locale]/quiz/create/_components/QuizCreateForm.tsx

```diff
@@ -0,0 +1,139 @@
+'use client';
+
+import React, { useState } from 'react';
+import { useRouter } from 'next/navigation';
+import { css } from '_panda/css';
+import { Button } from '@gitanimals/ui-panda';
+import { toast } from 'sonner';
+
+import QuizField from '@/app/[locale]/quiz/create/_components/QuizField';
+import QuizTextArea from '@/app/[locale]/quiz/create/_components/QuizTextArea';
+import Tabs from '@/components/Tabs/Tabs';
+import TabsList from '@/components/Tabs/TabsList';
+import TabsTrigger from '@/components/Tabs/TabsTrigger';
```

Comment:
같은 관심사라면 이렇게 내보내기 할 수 있지 않을까용???

```suggestion
import {Tabs, TabsList, TabsTrigger} from '@/components/Tabs';
```

지금처럼 하신 이유가 궁금해요!!

Reply:
shadcn의 방식이 이렇다고 생각했는데, 이제 보니 shadcn도 하나의 파일에서 import를 묶어 사용하네요.

```ts
import {
  Dialog,
  DialogContent,
  DialogDescription,
  DialogHeader,
  DialogTitle,
  DialogTrigger,
} from '@/components/ui/dialog';
```

이 부분 디자인시스템으로 넘길 때 고려해서 개발하겠습니다!

Reply:
`shadcn의 방식이 이렇다고 생각했는데, 이제 보니 shadcn도 하나의 파일에서 import를 묶어 사용하네요`

만약 shadcn이 내보내기를 나눠서 제공하는 형태라면, 나눠서 내보내려고 하셨다는 의도로 느껴지는데요

혹시 이유나 배경이 있을까용??

Reply:
제가 개발 철학을 잡을 때, 잘 갖춰진 라이브러리의 구조를 통해 기준을 잡아보려 했거든요.
shadcn이 트렌드에 민감하게 따라가고 구조도 잘 갖췄다고 생각해서 많이 따라보려 했습니다.

그런데 혜성님께서 그렇게 지적을 해주셔서 생각해보니, 맹목적으로 따를 필요는 없을 것 같다는 생각이 드네요. 코드 짤 때 더 고민해서 제 기준과 근거를 기반으로 짜보겠습니다. 정곡을 찔린 기분이네요ㅜㅜ 반성해봅니다. 좋은 피드백 감사합니다!